build.sbt

```diff
@@ -180,6 +182,7 @@ lazy val tapirInterop = project
       "com.softwaremill.sttp.tapir" %% "tapir-core"   % tapirVersion,
       "dev.zio"                     %% "zio-test"     % zioVersion % "test",
       "dev.zio"                     %% "zio-test-sbt" % zioVersion % "test",
+      "dev.zio"                     %% "zio-query"    % zqueryVersion,
```

Comment:
Another weirdness, I think I can remove, but my local sbt wasn't pulling in the transitive zio-query dependencies for some reason.